xml/Settings.xml

```diff
@@ -41,12 +41,13 @@
 					<texture colordiffuse="border_alpha">colors/black.png</texture>
 				</control>
 				<control type="image">
-					<left>95</left>
-					<top>10</top>
+					<centerleft>50%</centerleft>
```

Comment:
i dont think using percentage inside containers works properly unless you define some width for a parent group control inside the layout.